src/Algebra/Graph/Export/Dot.hs

```diff
@@ -57,12 +60,14 @@ data Style a s = Style
     -- ^ Attributes of a specific vertex.
     , edgeAttributes   :: a -> a -> [Attribute s]
     -- ^ Attributes of a specific edge.
+    , quoting :: Quoting
```

Comment:
It may be more future-proof to name this field `attributeQuoting`, since we might also add `labelQuoting :: Quoting` at some point (right now we always use double-quoting for labels).